images/bootstrap/create_bazel_cache_rcs.sh

```diff
@@ -24,9 +24,9 @@ package_to_version () {
 # look up a binary with which and return the debian package it belongs to
 command_to_package () {
     local binary_path
-    binary_path=$(readlink -f "$(which "$1")")
-    # `dpkg -S $package` spits out lines with the format: "package: file"
-    dpkg -S "$1" | grep "${binary_path}" | cut -d':' -f1
```

Comment:
This was unnecessarily complex and didn't always work for the case where `$1` was backed by a symlink (in our case python at `/usr/bin/python` -> `/usr/bin/python2.7`). 
The symlink is mentioned in both `python2.7-dev` and `python2.7-minimal`, but the hard entry at `/usr/bin/python2.7` resolves to `python2.7-minimal` as the owning package. 

`dpkg -S $file` works great for any of the actual fully resolved binaries (after readlink) so we can just use that instead and avoid https://github.com/kubernetes/test-infra/issues/8532